### DISCUSSION TITLE
Support for chronological reading order of complex book universes

### DISCUSSION THREAD

**Discussion by ElysiumFic** [advplyr/audiobookshelf#879] [Ideas]
A lot of expansive book universes have multiple series that overlap. For example, the various phases of the Dragonriders of Pern by Anne McCaffrey and her descendants. Some come after the original Dragonriders series, some come before, some books overlap each other chronologically (frex: The Harper Hall Trilogy interleaves with Dragonquest and The White Dragon.) Another example is the Dune universe as it encompasses both Frank Herbert's original books and the continuations by Brian Herbert and Kevin J. Anderson.

This doesn't include supplemental material found in anthologies. My favorite example of how convoluted these issues could be is David Weber's Honorverse, which is comprised (at the moment) of six series: Star Kingdom (prequel), Manticore Ascendant (quasi-prequel, but can stand alone, so whatevs), Honor Harrington, Worlds of Honor (six multi-author anthologies), Saganami Island, and Wages of Sin.

Ignoring the first half of the main series (Honor Harrington) as well as a number of shorts from the anthologies and the prequel series, here's what the latter half of the main would look like, if interwoven with the concurrent series and if each of the stories in the anthologies were slotted into its appropriate place chronologically:

>  _Honor Among Enemies_ (Honor Harrington #6)
>  "A Whiff of Grapeshot" (short story in _More than Honor_, Worlds of Honor #1)
>  "Changer of Worlds" (short story in _Changer of Worlds_, Worlds of Honor #3)
>  _In Enemy Hands_ (Honor Harrington #7)
>  _Echoes of Honor_ (Honor Harrington #8)
>  _Ashes of Victory_ (Honor Harrington #9)
>  "From the Highlands" (from _Changer of Worlds_)
>  "Nightfall" (from _Changer of Worlds_)
>  "Fanatic" (from _The Service of the Sword_, Worlds of Honor#4)
>  _War of Honor_ (Honor Harrington #10)
>  _Crown of Slaves_ (Wages of Sin #1 - for which "From the Highlands" and "Fanatic" provide necessary background)
>  _The Shadow of Saganami_ (Saganami Island #1)
>  _At All Costs_ (Honor Harrington #11)
>  _Storm from the Shadows_ (Saganami Island #2)
>  _Torch of Freedom_ (Wages of Sin #2)
>  _Mission of Honor_ (Honor Harrington #12)
>  _A Rising Thunder_ (Honor Harrington #13)
>  _Shadow of Freedom_ (Saganami Island #3)
>  _Cauldron of Ghosts_ (Wages of Sin #3)
>  _Shadow of Victory_ (Saganami Island #4)
>  _Uncompromising Honor_ (Honor Harrington #14)

 Seriously, these series overlap so tightly, sometimes the same scene can be found in a book from two or three different series, occasionally from a different character's viewpoint.)

So, while More Than Honor may be #1 in the Worlds of Honor series, "A Whiff of Grapeshot" would actually be Honor Harrington 6.1, or Honorverse ~#20 or so (Unless I counted the number of prequel books and shorts wrong.)

I've been trying to figure out HOW to handle these effectively for someone who wants to read these sorts of complicated universes in purely chronological order. Playlists can be written using timecodes to pinpoint a particular track, which would be useful for slotting a short story from an anthology in between novels, but very few players recognize them (I think VLC is one of the few, but it's been a number of years since I tried so I don't remember.)

There are plenty of MP4 atoms or ID3 frames that could store universe data if desired -- Work could be Universe, and Movement/Movement Number could be series/series part, or the TV Series/Season/Episode Name/Number atoms could also be creatively used. Or whatever. It's mostly the playback portion I can't work out, but it would be an awesome feature to somehow build in support for these sorts of series.

**Comment by advplyr** [comment]
Playlists is a feature that will be added.
One option is you can add another series to each book. There is no good folder structure for this but you can manually go in and add the universe series manually. Then on the series page you will see the full universe series and each child series.
